package.json

```diff
@@ -23,7 +23,7 @@
     "optimist": "~0.6.0",
     "q": "1.4.1",
     "saucelabs": "~1.3.0",
-    "selenium-webdriver": "3.0.1",
+    "selenium-webdriver": "3.5.0",
```

Comment:
Hi @vikerman @qiyigg 

Don't we want to go to 3.6.0 that's released a few weeks ago?

Reply:
It there any new features in 3.6.0 that you rely on?

Reply:
I was trying to search for the post of Simon Stewart but I couldn't find it. He said they didn't update the changelog (https://github.com/SeleniumHQ/selenium/blob/selenium-3.6.0/javascript/node/selenium-webdriver/CHANGES.md#vnext) but it was wise to update to 3.6.0 (on of the last versions before they are going to release version 4.0)

So if it's not to much effort I would advice to go to version 3.6.0

Reply:
OK, I will try it right now. If it is easy to upgrade to 3.6, I will upgrade protractor to 3.6 directly; otherwise, I 'll do it in a separate PR later on.